src/transformed_distribution.jl

```diff
@@ -39,7 +39,7 @@ bijector(d::Normal) = Identity{0}()
 bijector(d::MvNormal) = Identity{1}()
 bijector(d::PositiveDistribution) = Log{0}()
 bijector(d::MvLogNormal) = Log{1}()
-bijector(d::SimplexDistribution) = SimplexBijector{true}()
+bijector(d::SimplexDistribution, ::Type{Val{proj}} = Val{true}) where proj = SimplexBijector{proj}()
```

Comment:
IMHO, it's slightly nicer to hardcode usage of `SimplexBijector` with the correct `proj` value into the `link` and `invlink` definitions rather than changing the definition of `bijector` to accomodate the current definitions of `link` and `invlink`. Ideally, we'll just completely phase out the use of `link` and `invlink` at some point, and so I feel it's better to keep the `<: Bijector` interface consistent.

Therefore I'd suggest reverting this change back.

Reply:
I don't have a strong opinion on this, so I just reverted the change. I agree that the change introduced an unfortunate inconsistency in the `bijector` interface. The main motivation for it was to be able to obtain an instance of `SimplexBijector{false}` with `bijector` - that's not possible anymore in the updated version. However, I guess it's fine to just use the default `SimplexBijector{true}()` since users can still obtain the non-standard `SimplexBijector{false}()` without `bijector`.